packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -1531,11 +1550,16 @@ export class ApexDebug extends LoggingDebugSession {
     const type: ApexDebuggerEventType = (ApexDebuggerEventType as any)[
       message.sobject.Type
     ];
+    this.log(
+      TRACE_CATEGORY_STREAMINGAPI,
+      `handleEvent: received ${JSON.stringify(message)}`
+    );
     if (
       !this.mySessionService.isConnected() ||
       this.mySessionService.getSessionId() !== message.sobject.SessionId ||
       this.myStreamingService.hasProcessedEvent(type, message.event.replayId)
     ) {
+      this.log(TRACE_CATEGORY_STREAMINGAPI, `handleEvent: ignoring`);
```

Comment:
I'm not sure what I'd do with this information if I saw this in Debug Console.

Reply:
Would it help if the event itself is printed? It should be printed just the line above, though.